pgsync/elastichelper.py

```diff
@@ -38,16 +43,29 @@ def __init__(self):
         host = 'localhost', port = 9200
         """
         url = get_elasticsearch_url()
-        self.__es = Elasticsearch(
-            hosts=[url],
-            timeout=ELASTICSEARCH_TIMEOUT,
-            verify_certs=ELASTICSEARCH_VERIFY_CERTS,
-            use_ssl=ELASTICSEARCH_USE_SSL,
-            ssl_show_warn=ELASTICSEARCH_SSL_SHOW_WARN,
-            ca_certs=ELASTICSEARCH_CA_CERTS,
-            client_cert=ELASTICSEARCH_CLIENT_CERT,
-            client_key=ELASTICSEARCH_CLIENT_KEY,
-        )
+        if(AWS_HOSTED):
```

Comment:
This is great. However, I'm worried this might not apply every time you are using AWS infrastructure. I am already using this in an AWS backend service without having to this.
I feel the Elasticsearch client should encapsulate all this?

Reply:
@toluaina How is your elasticsearch cluster hosted? If you are using their fully managed service, signing all the HTTP requests is mandatory.

https://docs.aws.amazon.com/elasticsearch-service/latest/developerguide/es-request-signing.html#es-request-signing-python